src/App/Provider.php

```diff
@@ -206,7 +206,7 @@ public function register(): ContainerBuilder
             ->addArgument(new Reference(EventDispatcher::class))
             ->addArgument(new Reference(Configuration::class));
 
-        $listenerProvider = new \PhpAT\App\EventListenerProvider($this->builder, $this->output);
+        $listenerProvider = new() \PhpAT\App\EventListenerProvider($this->builder, $this->output);
```

Comment:
:warning: This automatic code style correction of phpcs makes no sense at all!

Reply:
That's weird... and actually breaks the execution in php. Did phpcbf do this? wtf
Please if you find some info or explanation regarding this let me know 😕